tests/test_signals.py

```diff
@@ -0,0 +1,155 @@
+# coding: utf-8
+"""Tests for :mod:`nailgun.signals`."""
+# pylint: disable-all
+from sys import version_info
+from mock import patch
+from nailgun import entities, config, signals, client
+
+if version_info < (3, 4):
+    from unittest2 import TestCase
+else:
+    from unittest import TestCase
+
+ORG_DATA = {"id": 1, "domain": "http://example.com"}
+
+
+class FakeResponse(object):
+    '''Faking here cause we only need to test signals'''
```

Comment:
We use `"` for docstring, can you please follow the code style? Same applies for other docstrings.